tests/filesystem_model_test.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from gcp_census.model.filesystem_model_provider import FilesystemModelProvider
+
+
+class TestFilesystemModelProvider(unittest.TestCase):
+
+    def test_should_return_at_least_3_groups(self):
```

Comment:
Not sure about this, I would like to discuss. If you change bq_schemas conf then you need to fix tests.

Reply:
Previously all changes in schema number required to update the tests, now we are only testing the provider to return required minimum